admin.py

```diff
@@ -92,6 +93,21 @@ def execute(self):
         command.stamp(alembic_cfg, "head")
 
 
+class Alembic(Command):
+    """ Runs arbitrary alembic command. """
+
+    needs_session = False
+
+    def setup_parser(self, parser):
+        parser.add_argument('args', nargs='*',
+                            help="Arguments passed to alembic")
+
+    def execute(self, args):
+        exe = '/usr/bin/python3-alembic' if six.PY3 else '/usr/bin/alembic'
```

Comment:
I think using `sys.executable` would be better

Reply:
Why? Can you elaborate?

Reply:
Or executing its main directly in the same interpreter:
````python
        from alembic.config import main as alembic_main
        config_path = get_config('alembic.alembic_ini')
        argv = ['-c', config_path] + args
        alembic_main(prog='koschei-admin alembic', argv=argv)
````

Reply:
> Why?

IMO the executable split between alembic and python3-alembic is an implementation detail of our packaging

Reply:
I'll refactor it to call alembic module within the same Python interpreter.